ppr-ui/src/composables/useInputRules.ts

```diff
@@ -55,13 +55,23 @@ export const useInputRules = () => {
   }
 
   const isEmail = (): Array<Function> => [
-    (v: string) => !!v || 'Email address is required',
+    (v: string) => !!v || 'Enter an email address',
     (v: string) => {
       const pattern = /^(([^<>()[\]\\.,;:\s@"]+(\.[^<>()[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/
-      return pattern.test(v) || 'Valid email is required'
+      return pattern.test(v) || 'Enter a valid email address'
     }
   ]
 
+  const isPhone = (
+    maxDigits: number = null,
+    customMsg: string = null
+  ): Array<Function> => {
+    return [
+      v => ((v && maxDigits) ? v.length >= maxDigits : true) || 'Minimum 10 characters',
+      v => (v ? /^\d+$/g.test(v.replace('(', '').replace(') ', '').replace('-', '')) : true) || `${customMsg || 'Must contain numbers only'}`
```

Comment:
Just my feedback, if we enter `(123) A11-2233`, the message 'Must contain numbers' is confusing. Maybe more of a question to UX team...

Reply:
I have made the change to: "Enter numbers only".  This is base on UX Design comments on ticket

Reply:
Changed to "Enter numbers only".  This is based on UX review